src/app/client/src/app/modules/shared/components/material-auto-complete/material-auto-complete.component.ts

```diff
@@ -34,6 +34,8 @@ export class MaterialAutoCompleteComponent implements OnChanges {
     this.dropDownSelectedShow();
   }
 
+  @ViewChild("autocompleteInput", { static: false }) searchField: ElementRef;
```

Comment:
I think the static:false is not needed in angular 9 please check

Reply:
Yes, Updated